codacy/templates/fluentd/fluentd-conf.yaml

```diff
@@ -91,7 +92,7 @@ data:
         flush_mode interval
         # HACK: Retry writting the logs forever to recover from network issues
         #       This helps when minio/s3 is only available after fluentd starts
-        retry_forever true
+        retry_timeout {{ .Values.fluentdoperator.flushTimeout }}
```

Comment:
This will expire erroneous logs instead of keeping them in the var.log.container buffer forever